README.md

```diff
@@ -160,21 +143,104 @@ try {
+import bPromise, { b } from './b.mjs'
+import cPromise, { c } from './c.mjs'
+
+Promise.all([aPromise, bPromise, cPromise]).then(() => {
+
+console.log(a, b, c);
+
+});
+```
+
+Modules `a.mjs`, `b.mjs`, and `c.mjs` would all execute in order up until the first await in each of them; we then wait on all of them to resume and finish evaluating before continuing.
+
 ### FAQ
 
 #### Isn't top-level `await` a footgun?
 
-If you have seen [the gist](https://gist.github.com/Rich-Harris/0b6f317657f5167663b493c722647221) you likely have heard this critique before. My hope is that as a committee we can weigh the pros / cons of the various approaches and determine if the benefits of the feature outweigh the risks.
+If you have seen [the gist](https://gist.github.com/Rich-Harris/0b6f317657f5167663b493c722647221) you likely have heard this critique before.
+
+Some responses to some of the top concerns here:
+
+##### Will top-level `await` cause developers to make their code block longer than it should?
+
+It's true that top-level `await` gives developers a new tool to make their code wait. Our hope is that proper developer education can ensure that the semantics of top-level `await` are well-understood, so that people know to use it just when they intend that importers should block on it.
+
+We've seen this work well in the past. For example, it's easy to write code with async/await that serializes two tasks that could be done in parallel, but a deliberate developer education effort has popularized the use of `Promise.all` to avoid this hazard.
+
+##### Will top-level `await` encourage developers to use `import()` unnecessarily, which is less optimizable?
+
+Many JavaScript developers are learning about `import()` specifically as a tool for code splitting. People are becoming aware of the relationship between bundling and multiple requests, and learning how to combine them for good application performance. Top-level `await` doesn't really change the calculus--using `import()` from a top-level `await` will have similar performance effects to using it from a function. As long as we can tie top-level `await`'s educational materials into the existing knowledge of that performance tradeoff, we hope to be able to avoid counterproductive increases in the use of `import()`.
+
+#### What exactly is blocked by a top-level `await`?
+
+When one module imports another one, the importing module will only start executing its module body once the exporting module's body has finished executing. If the exporting module reaches a top-level await, that will have to complete before the importing module's body starts executing.
+
+#### Why doesn't top-level `await` block the import of an adjacent module?
+
+If one module wants to declare itself dependent on another module, for the purposes of waiting for that other module to complete its top-level `await` statements before the module body executes, it can delcare that other module as an import.
```

Comment:
```suggestion
If one module wants to declare itself dependent on another module, for the purposes of waiting for that other module to complete its top-level `await` statements before the module body executes, it can declare that other module as an import.
```